uk.co.compucorp.civicrm.hremails/email-templates/src/pages/sickness-request.civicrm.html

```diff
@@ -0,0 +1,64 @@
+---
+layout: leave-absences
+subject: CiviHR Sickness Request
+request-type: Sickness
+---
+
+<div class="request request-sickness">
+  <row class="collapse">
+    <columns>
+      {{> callout-header title="Leave Request Type"}}
+      <callout class="request-data">
+        {{> request-data key="Status" value="{$leaveStatus}"}}
+        {{> request-data key="Staff Member" value="{contact.display_name}"}}
+
+        {if $leaveRequest->from_date eq $leaveRequest->to_date}
+          {{> request-data key="Date" value="{$fromDate|truncate:10:''|crmDate} {$fromDateType}"}}
+        {else}
+          {{> request-data key="From Date" value="{$fromDate|truncate:10:''|crmDate} {$fromDateType}"}}
+          {{> request-data key="To Date" value="{$toDate|truncate:10:''|crmDate} {$toDateType}"}}
+        {/if}
+        <hr>
+        {{> request-data key="Additional Details" value=""}}
```

Comment:
this was done simply for layout purposes and to reuse what we have instead of adding some exception to the style. Basically additional details has the same look of any label in a label/value couple, so i reused the request-data partial and kept the value empty